notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>What's pct and pct_lower and how are we going to define them? Why do we select 11 bins?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
pct and pct_lower is basically a measure of percent change, we start from 0.01 up to 0.99. Ideally covering all the values from 0-100%
We initially set up 11 bins according to the number of classes, but changing it to a different number doesn't affect our chi-square values.